packages/backend/src/models/GroupsModel.ts

```diff
@@ -221,4 +223,54 @@ export class GroupsModel {
             organizationUuid: updatedGroup.organization_uuid,
         };
     }
+
+    async addUserToGroupsIfExist({
+        userUuid,
+        userId,
+        groups,
+        organizationUuid,
+    }: {
+        userUuid?: string;
+        userId?: number;
+        groups: string[];
+        organizationUuid: string;
+    }) {
```

Comment:
Had to read the logic to understand why we supported the user uuid and id. 

Suggestion:
```{
        userId: number;
        groups: string[];
        organizationUuid: string;
    } || {
        userUuid: string;
        groups: string[];
        organizationUuid: string;
    }
```

Reply:
makes sense, thanks! adding now